N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
yes, the project is called N3.js but does it actually parse the N3 syntax defined here?
  http://www.w3.org/TeamSubmission/n3/
Ie does it do graphs, rules, and support the path notations '.' and '^' ?